script/api/validate_spec

```diff
@@ -2,26 +2,21 @@
 
 require 'pathname'
 require 'tempfile'
-require 'psych'
-require 'yaml'
 
-load 'lib/api/open_api.rb'
-
-assembled_path = "./tmp/oas-generated.yml"
-spec_path = Pathname(__dir__).join("../../docs/api/apiv3/openapi-spec.yml")
-full_spec = API::OpenAPI.send(:assemble_spec, spec_path).to_yaml
+openapi_yaml_spec_path = "./tmp/oas-generated.yml"
+full_spec = `#{Pathname(__dir__).join('spec')} --format yaml`
```

Comment:
Its's a refactoring. I just removed some duplication.

`spec` is a script that already exists and outputs the bundled openapi spec. At first I was commenting out the `FileUtils.rm` to get the bundled file, and then I saw the `spec` script, and thought that if the first script was calling the other, I would have discovered the existence of that `spec` script and I could use it without modifying the `validate_spec` script.

Since then I discovered that `npx @redocly/openapi-cli bundle docs/api/apiv3/openapi-spec.yml --output bundled.yaml` does exactly the same, without needing some code from us. Not sure if I should change it again.

Reply:
ahhhhhh, `spec` is the script, not the directory.